.github/workflows/coding-style-check.yml

```diff
@@ -36,10 +38,24 @@ jobs:
         # When updating the extension list, remember to update
         # Jamulus.pro's CLANG_SOURCES as well and the paths: in 'on:' list above.
         extensions: 'cpp,h,mm'
+
+  check-bash-style:
+    name: Verify shell script (Bash) coding style
+    runs-on: ubuntu-latest
     # shellcheck is already pre-installed on ubuntu-latest
+    steps:
     - name: Check .sh with shellcheck
       run: find -name '*.sh' -not -path './libs/*' -exec shellcheck --shell=bash {} +
     - name: Install shfmt
       run: sudo snap install shfmt
     - name: Check .sh with shfmt
       run: shfmt -d .
+
+  check-python-style:
+    name: Verify Python coding style
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install pylint
+      run: pip install --user "pylint < 3.0"
```

Comment:
We might want to have an automatic update here. But that's not in scope of this PR.